apps/tutorial-servo-can-control/TutorialServo.cpp

```diff
@@ -0,0 +1,45 @@
+
+#include "mbed.h"
+#include "TutorialServo.h"
+
+#include <iostream>
+using namespace std;
+
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0,
+float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2)
+{
+    //declares this pin as pwm using pwmout.
+    PwmOut servoPwmOut(servoPin);
+
+    //set the PWM period for that pin to 20ms
+    servoPwmOut.period_ms(20);
+
+
+    while(1)
+    {
+        servoPwmOut.pulsewidth_ms(minPulsewidthInMs);
+        servoPwmOut.pulsewidth_ms(maxPulsewidthInMs);
+    }
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees)
+{
+   
+    m_servoRangeInDegrees = degrees;
```

Comment:
What you are doing here is changing the member variable, this isn't going to change the angle of the motor at all. What you want to do is to call the method of setting the pulsewidth on the servoPwmOut i.e including the following line of code instead:

m_servoPwmOut.pulsewidth_ms(some_argument);

Think about what should the argument for this function be and dont hesitate to hit me up on discord for more help.